projects/mercury/src/metadata/views/MetadataViewTableContainer.js

```diff
@@ -227,18 +317,28 @@ export const MetadataViewTableContainer = (props: MetadataViewTableContainerProp
                     {renderMessages()}
                     {renderMetadataViewTable()}
                 </TableContainer>
-                <TablePagination
-                    rowsPerPageOptions={[5, 10, 25, 100]}
-                    component="div"
-                    count={count && isNonEmptyValue(count.count) ? count.count : -1}
-                    rowsPerPage={rowsPerPage}
-                    page={page}
-                    onPageChange={handleChangePage}
-                    onRowsPerPageChange={handleChangeRowsPerPage}
-                    className={classes.tableFooter}
-                    labelDisplayedRows={(d) => labelDisplayedRows({...d, countIsLoading: loadingCount})}
-                    ActionsComponent={TablePaginationActions}
-                />
+                <div className={classes.footerButtonDiv}>
+                    <Button
```

Comment:
I think the button would look nicer with extended width:
![Screenshot from 2022-09-23 12-49-44](https://user-images.githubusercontent.com/20908736/191945280-37c99c94-0d11-4e41-9c81-a8588b0d334c.png)

Reply:
And maybe it could be disabled if selection=0?

Reply:
yes, that's a good idea!